tests/functional/delete_runs/delete_runs_test_files/project_info.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "simple",
```

Comment:
Do we need a separate test project for this? There is already one with some bugs in it.